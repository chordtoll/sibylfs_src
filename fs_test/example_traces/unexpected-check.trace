# processing file 'example_traces/os_trace1-int.trace' ...
        @type trace
     1: open /ooOOoo [] 0o000
        Tau
        RV_num(3)
