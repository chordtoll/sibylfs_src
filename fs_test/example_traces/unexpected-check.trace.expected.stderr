
Error:      5: RV_num(3)
   unexpected results: RV_num(3)
   allowed are only: ENOENT
   continuing execution with ENOENT
